{{cookiecutter.project_slug}}/clients/vue3/src/views/PageNotFound.vue

```diff
@@ -1,11 +1,9 @@
 <template>
-  <div class="page-not-found">
-    <h1 class="page-not-found__error">404</h1>
+  <div class="flex flex-col items-center">
+    <h1 class="text-7xl font-extrabold">404</h1>
     <img alt="Project Logo" src="../assets/logo.png" />
-    <h2 class="page-not-found__header">Oops!<br />We couldn't find the page you're looking for.</h2>
-    <p class="page-not-found__route">
-      Click <router-link to="/">here</router-link> to return to the home page.
-    </p>
+    <h2 class="text-xl font-bold mb-4">We couldn't find the page you're looking for.</h2>
+    <router-link to="/" class="text-xl underline">Return to the home page.</router-link>
```

Comment:
**A11y:** Removed the "click here" text as that's not helpfully descriptive for screen-reader users. We want to make sure the whole link [text] is descriptive as to what the link directs the user to (and not just have the link say "here").

Reply:
👍🏻 Excellent. And for sighted users it's going to be a much easier target to click, too!

Reply:
Yup, I know I've been notoriously bad about doing the "Click **here**" method for links, so trying to undo some of that now!

Reply:
Haha, well, if you want to go the extra mile and impress people at parties, [**there is also Fitt's Law, which provides a mathematical argument in favor of wide links.**](https://en.wikipedia.org/wiki/Fitts%27s_law)